.github/workflows/nightly-dkg.yml

```diff
@@ -1,5 +1,5 @@
-# This runs long wait DKG test once a day at 12:00 noon.
-name: long wait dkg
+# This runs DKG tests that assert edge cases and long execution times once a day at 12:00 noon.
+name: nightly dkg
```

Comment:
fun: It is more of a `day dkg` test since it happens at noon, not night 🫣

Please ignore this comment